Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalVaultRequest.h

```diff
@@ -8,11 +8,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface BTPayPalVaultRequest : BTPayPalRequest
 
-/**
- Optional: Display a custom description to the user for a billing agreement.
-*/
-@property (nonatomic, nullable, copy) NSString *billingAgreementDescription;
```

Comment:
It's a little weird that this class doesn't have any public properties now that `billingAgreementDescription` has been moved to `BTPayPalRequest`. But I think it still makes sense to keep the separate class since it makes our public API a little nicer and we might have things to add here in the future.